### ISSUE TITLE
CLI and curses not using tracker_update_wait

### ISSUE THREAD

**Issue by thnikk** [z411/trackma#354]
Setting "Wait before update" in GTK or manually setting tracker_update_wait(_s) work in GTK but not in the cli or curses. Instead, it always says it will update in 1140 seconds. I'd like to be able to use this on my headless server but I either have to x forward (using GTK) or not pause during the episode (using cli/curses) to get it to work. 

Tested on v0.7.4

**Comment by z411** [comment]
It shows in how much time it will update, but the actual timer should display in the bottom right corner. In the CLI you can use the `tracker` command to see the timer.

**Comment by thnikk** [comment]
It still gives the same value (same in the bottom right with curses):

Tracker (Plex): Will update Comic Girls 10 in 1140 seconds
thnikk [anilist] (anime) watching >> tracker
- Tracker status -
State: Playing
Filename: [HorribleSubs] Comic Girls - 10 [720p].mkv
Timer: 1140
Show: Comic Girls
Episode: 10

Interestingly, it seems like it could be determined by the episode length (Alice or Alice is only 3 mins per ep.)

Tracker (Plex): Will update Comic Girls 10 in 1140 seconds
Tracker (Plex): Player was closed before update.
Tracker (Plex): Will update Alice or Alice: Siscon Niisan to Futago no Imouto 10 in 180 seconds
Tracker (Plex): Player was closed before update.
Tracker (Plex): Will update Comic Girls 10 in 1140 seconds
Tracker (Plex): Player was closed before update.

**Comment by llwu** [comment]
You need in config.json,

“plex_obey_update_wait_s”: true

otherwise it’ll use 80% of the episode length (supposedly an approximation of the episode length less OP and ED)

**Comment by Hairo** [comment]
Yeah, it's weird that it only works in the Gtk/Qt GUIs, what is `plex_obey_update_wait_s` set to in your config.json?

**Comment by thnikk** [comment]
Yup, definitely just user error. I recently transitioned back to Windows on my desktop which is why I went from GTK to CLI, so the very obvious checkbox became a slightly less obvious line in the config file. Setting plex_obey_wait_s to true fixed it.

It is still strange, though. On a fresh install, it would use the set wait value regardless of the obey setting, but only in the GUI. After toggling it on and off again, GTK now shows 1140 with the setting off (which is to be expected.) 

Thanks a lot for the help and sorry for being blind. I'll close this since my specific problem was solved.
